docs/reference/git-tables.md

```diff
@@ -39,12 +39,15 @@ SELECT * FROM commits('/some/path/to/repo')
 -- clone a remote repo and use it
 SELECT * FROM commits('https://github.com/mergestat/mergestat')
 
--- use the default repo, but provide an alternate branch
+-- use the default repo, but provide an alternate branch/ref
+-- list available refs and branches with `SELECT * FROM refs('https://github.com/mergestat/mergestat')`
 SELECT * FROM commits('', 'some-ref')
```

Comment:
Note: this example currently returns `failed to open "/": must be an http(s) repo` on the public workspace

Reply:
Other examples could be `SELECT * FROM commits('https://github.com/mergestat/mergestat', 'v0.1.0')`

Reply:
Thanks for that! FYI, if you click the `Query Settings` button in the web UI

![image](https://user-images.githubusercontent.com/57259/151236694-a20a622e-7d8b-4515-bebd-7a06d0796783.png)

you'll be able to set the `Default Repository` which allows you to avoid typing out a full HTTPs repo path:

![image](https://user-images.githubusercontent.com/57259/151236758-bce41b80-08d4-4eda-8487-57123f6fb3a4.png)